From bd27f269bde4fee617506f5f869842cc8ceab827 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Sun, 16 Jan 2022 19:49:21 -0500
Subject: [PATCH] Fix #121 (failure to compile with vala 0.55.1)

Make create_exec_filename public so it is not less accessible than
OPTIONS, the value of which contains it.
---
 src/Application.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Application.vala b/src/Application.vala
index 118e88d..45212a1 100755
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -26,7 +26,7 @@ public class AppEditor.Application : Gtk.Application {
         return Gtk.check_version (3, 22, 0) == null;
     }
 
-    private static string? create_exec_filename;
+    public static string? create_exec_filename;
 
     private MainWindow? window = null;
 
